uid-pages/page-admin-profile-list/src/adminProfileListBonitaV1.json

```diff
@@ -11176,7 +11176,7 @@
                                                 },
                                                 "label" : {
                                                   "type" : "interpolation",
-                                                  "value" : "{{$item.displayName}}"
+                                                  "value" : "{{editMembershipsMappingCtrl.getRoleDisplayName($item)}}"
```

Comment:
Same as for the groups

Reply:
Same as a group, 
name = ceo
display name = Chief Executive Officer